providers/openebs/installers/operator/0.6/ansible/openebs.yaml

```diff
@@ -9,24 +9,43 @@
 
   tasks:
     - block:
-          - name: Add path to openebs storage pool
+
+          - name: Downloading openebs-operator.yaml
+            get_url:
+              url: "{{ openebs_operator_link }}"
+              dest: "{{ playbook_dir }}/openebs-operator.yaml"
+
+
+          - name: Updating maya api server image
             replace:
-              dest: sp.yaml
-              regexp: '/mnt/openebs'
-              replace: "{{ lookup('env','mountPath') }}"           
+              path: "openebs-operator.yaml"
+              regexp: "image: openebs/m-apiserver.*"
+              replace: "image: {{ lookup('env','MAYA_APISERVER_IMAGE') }}"
+            when: lookup('env','MAYA_APISERVER_IMAGE') | length > 0
 
-          - name: Applying openebs operator
-            shell: \{{ kubeapply }} apply -f {{ openebsOperatorLink }} 
-            args:
-              executable: /bin/bash
+          - name: Updating openebs provisioner image
+            replace:
+              path: "openebs-operator.yaml"
+              regexp: "image: openebs/openebs-k8s-provisioner.*"

```

Comment:
Had seen earlier reports on wildcards use in regex_replace. Can you please paste a sample o/p of these tasks ?

Reply:
suppose I have a environment variable OPENEBS_PROVISIONER_IMAGE set to testimage then the playbook will replace the image name of openebs-provisioner to testimage. Like this :
```        imagePullPolicy: Always
        image: testimage```